tests/InputFiles/Xcts/KerrSchild.yaml

```diff
@@ -60,8 +60,9 @@ DomainCreator:
         InitialValues:
           Mass: *KerrMass
           Spin: *KerrSpin
-      TranslationMap:
-        InitialValues: [*Center, [0., 0., 0.]]
+      RotationMap: None
+      ExpansionMap: None
+      TranslationMap: None
```

Comment:
Do you still need the large timeout when all of these are `None`?

Reply:
Yeah, I think it's mostly because of the extra radial partition. We could modify de example to not use the shape map (?)

Reply:
We could use the shape time dependence instead of the hard coded shape maps. It'd look like this (with a couple extra yaml anchors you need to add):

```yaml
    TimeDependentMaps:
      Shape:
        InitialTime: 0.0
        LMax: 10
        Mass: *KerrMass
        Spin: *KerrSpin
        Center: *KerrCenter
        InnerRadius: *InnerRadius
        OuterRadius: *OuterRadius
```

@nilsvu Does that seem ok?

Reply:
Why did you have to add the extra radial partition? Would be better to keep the domain the same here (log map around the horizon seems important).

Reply:
So with the hardcoded maps, the shape map cannot extend into the region where the expansion/translation map are transitioning to zero because the shape map changes the radius of coordinates and in the transition region the exp/trans map make an assumption that the coords they are using are the grid coordinates. Not some distorted coordinates. The easiest way to avoid this is to just require at least one partition. The shape map is in the innermost shell, and the exp/trans transition is in the outermost shell.

For this test though, if we use the shape time dependence, we can avoid an extra partition and leave it as it was originally because there is no exp/trans maps at all. I think this is the easiest solution for leaving the test as unchanged as possible. And I believe it'll avoid the long timeouts

Reply:
I went back to the old test and implemented @knelli2 suggestion for the shape map